deploy/helm/sumologic/conf/instrumentation/traces.sampler.conf.yaml

```diff
@@ -15,12 +15,6 @@ exporters:
     compression: gzip
 {{- end }}
 
-  ## Following generates verbose logs with span content, useful to verify what
```

Comment:
I have removed this comment as it is not accessible to users anymore (it was before when it was in `values.yaml`).